src/bra.cpp

```diff
@@ -287,6 +379,10 @@ int main(int argc, char* argv[])
                         fs::perm_options::add);
 #endif
 
+        // remove TMP SFX FILE
+        if (!fs::remove(g_out_filename))
+            cout << format("WARN: unable to remove temporary file {}", g_out_filename.string()) << endl;
+
```

Comment:
this is not optimal: it requires twice the disk space to create an SFX ... 

find an improvement.

Reply:
#ref 18